.prettierrc.json

```diff
@@ -1,3 +1,4 @@
 {
+  "printWidth": 120,
```

Comment:
Open to feedback on this exact number, I essentially randomly chose it. I'm hoping that's a better compromise for test fixture geojson readability, among other things. (I think the default was 80?)